client/src/components/backend/Dialog/Confirm.js

```diff
@@ -72,3 +91,8 @@ export default class DialogConfirm extends PureComponent {
   }
 
 }
+
+DialogConfirm.contextTypes = {
```

Comment:
Can't these be added as static properties on the class? That would be more in keeping with what we're doing elsewhere with propTypes, etc.